### PR TITLE
SequenceNumber_t improvements [6359]

### PR DESCRIPTION
**Pull request by MiguelCompany**
This PR improves SequenceNumber.h
* Applies coding style 
* Removes LLONG_MAX
* Improves performance on some operators

### CODE REVIEW

File: include/fastrtps/rtps/common/SequenceNumber.h [hunk 1]
```diff
@@ -84,22 +93,18 @@ struct RTPS_DllAPI SequenceNumber_t
 
 
     //! Increase SequenceNumber in 1.
-    SequenceNumber_t& operator++() noexcept
+    SequenceNumber_t& operator ++() noexcept
     {
-        if(low == UINT32_MAX)
```
Comment (raquelalvarezbanos) [inline include/fastrtps/rtps/common/SequenceNumber.h]: Can the corresponding header be removed?

Comment (MiguelCompany) [reply include/fastrtps/rtps/common/SequenceNumber.h]: done!

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Linux/437/)
* Mac [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Mac/395/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Windows/435/)

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Linux/440/)
* Mac [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Mac/398/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Windows/438/)

**Comment by raquelalvarezbanos** [comment]
Windows failures are:
* projectroot.test.blackbox.BlackBox.PubSubKeepAll
* projectroot.test.blackbox.BlackBox.BuiltinAuthenticationAndCryptoPlugin_reliable_all_data300kb_mix
* projectroot.test.communication.SimpleCommunicationSecureBestEffort

which we've seen failing before.

**Pull request merged**